src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -138,43 +136,44 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
 
   const adminLabel = (item: any): JSX.Element => {
     if (item.isAdmin) {
-      return <div style={{ textAlign: 'left', paddingLeft:'10px' }}>
-        <Label><FormattedMessage id='Yes' /></Label>
+      return <div style={{paddingRight: '28px', textAlign: 'center' }}>
+        <span>{translateMessage('Yes')}</span>
       </div>;
     } else {
-      return <div style={{ textAlign: 'left', paddingLeft: '10px' }}>
-        <Label><FormattedMessage id='No' /></Label>
+      return <div style={{paddingRight: '28px', textAlign: 'center' }}>
+        <span>{translateMessage('No')}</span>
       </div>;
```

Comment:
Can we have the styles defined? they are identical, they can be defined in a style definition above where they are used

Reply:
I have lifted all the styles to the .styles file. Nice observation